storage-access.bs

```diff
@@ -272,9 +270,9 @@ ISSUE: [since this is UA-defined, does it make sense to follow-up separately wit
 
 <h3 id="navigation">Changes to navigation</h3>
 
-Before changing the [=current entry=] of a [=session history=], run the following steps:
+Before changing the current entry of a session history, run the following steps:
 
-1. Let |doc| be [=current entry=]'s {{Document}}.
```

Comment:
I have no idea, but let me know if they get in the way of merging. I can rebase to try to remove them.

Reply:
I think that's because it's not rebased against main, but there are no conflicts as these are the same changes.

Squashing and rebasing against main might be slightly safer.

Reply:
Resolved.